packages/backend/src/workers/provider/OpenVINO.ts

```diff
@@ -0,0 +1,296 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+import type {
+  ContainerCreateOptions,
+  ContainerProviderConnection,
+  DeviceRequest,
+  ImageInfo,
+  MountConfig,
+} from '@podman-desktop/api';
+import type { InferenceServerConfig } from '@shared/models/InferenceServerConfig';
+import { InferenceProvider } from './InferenceProvider';
+import { getModelPropertiesForEnvironment, getMountPath } from '../../utils/modelsUtils';
+import { DISABLE_SELINUX_LABEL_SECURITY_OPTION } from '../../utils/utils';
+import { LABEL_INFERENCE_SERVER } from '../../utils/inferenceUtils';
+import type { TaskRegistry } from '../../registries/TaskRegistry';
+import { type InferenceServer, InferenceType } from '@shared/models/IInference';
+import { GPUVendor, type IGPUInfo } from '@shared/models/IGPUInfo';
+import { VMType } from '@shared/models/IPodman';
+import type { PodmanConnection } from '../../managers/podmanConnection';
+import type { ConfigurationRegistry } from '../../registries/ConfigurationRegistry';
+import { openvino } from '../../assets/inference-images.json';
+import { existsSync } from 'node:fs';
+import { writeFile } from 'node:fs/promises';
+import type { ModelInfo } from '@shared/models/IModelInfo';
+import type { ModelsManager } from '../../managers/modelsManager';
+
+export const SECOND: number = 1_000_000_000;
+
+const CONFIG_FILE_NAME = `config-all.json`;
+
+const GRAPH_CONTENT = `input_stream: "HTTP_REQUEST_PAYLOAD:input"
```

Comment:
feel like this should be in an `packages/backend/src/assets/openvino-config.txt` and imported as `?raw`?